### ISSUE TITLE
Resize Windows options

### ISSUE THREAD

**Issue by c-sanchez** [RamonUnch/AltDrag#53]
Can be nice to have the features from Sizer by Brian Apps in AltDrag.
Options such as setting and saving predefined window size values, being able to view window sizes, etc.

Check Sizer program
http://www.brianapps.net/sizer/

**Comment by RamonUnch** [comment]
Interesting, I did not know about this program, thanks for the info.
There are a lot of small features, some of which I might implement into AltDrag.
For example, I could easily display the windows size at the top of the Action Menu.

However I would not want to modify too much AltDrag when this program works fine alongside it.
I do not like to re-write code that already works fine and I do not see a problem with the sizer program
If you find any annoying interaction bugs, of course report them.

So yes it is possible that some features will be implemented but only if they integrate well within AltDrag and if I have time to implement them and if it is not hard to implement.

**Comment by Ichisich** [comment]
You could combine window dimensions with the already implemented _**centre window on screen**_ feature.
Similar to UseZone, let a test window centre and resize it to your desired dimensions and set it as default centre size.
Or alternatively, put in the (width/height) dimension in the .ini file, in which case (0,0) would be keep the size as is and (xx,0) or (0,xx) only scale one axis.

e.g. (1280,720) in 1080p would look like this:
![Screenshot 2021-09-19 105412](https://user-images.githubusercontent.com/31080003/133921461-5ecc62fc-f2dd-4b55-b4f0-f571ea39e2f4.png)

**Comment by RamonUnch** [comment]
Yes for sure it is not complex or hard to implement, the idea would be to a list of configurable preset sizes to the action menu. It is not necessary to center the window, its position can be left the same.
I will not implement it right now however because the sizer program works quite fine and is more powerful than what I would do. Sizer can be used alongside AltSnap so it is not a priority but it will be implemented at some point, because it seems to be a desired feature and I completely understand people not liking to have loads of background programs, I myself minimize the amount.
